### PR TITLE
DHP-374 Rake task to reset all Veteran fitbit device records

### PR DESCRIPTION
**Pull request by yonidoronpeters**
## Description of change
* Create rake task to set all connected Veteran Fitbit records to `false`. **This task should never be used in Production**
* Move Create Devices rake task to the `dhp_connected_devices` module where it belongs

### CODE REVIEW

**Review by LindseySaari — DISMISSED** [review]

**Review by reblg — APPROVED** [review]

**Comment by yonidoronpeters** [comment]
@LindseySaari would you mind reapproving the PR please?

File: modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module DhpConnectedDevices
+  module DataMigrations
+    module ResetAllVeteranFitbitRecords
+      module_function
+
+      def run
+        # Warning! this will reset ALL veteran records to show they do not have a fitbit connected to their account
```
Comment (mchaffee-tw) [inline modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records.rb:9]: Did we create a card to make this only run in lower environments?

**Review by mchaffee-tw — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]

File: modules/dhp_connected_devices/lib/tasks/dhp_reset_all_veteran_fitbit_device_records.rake [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records'
+
+namespace :data_migration do
+  desc 'Warning: This task resets ALL Veteran records to show they do not have a Fitbit device connected'
+  task dhp_reset_all_veteran_fitbit_device_records: :environment do
+    raise 'This task should not be run in production' if Settings.vsp_environment == 'production'
```
Comment (yonidoronpeters) [inline modules/dhp_connected_devices/lib/tasks/dhp_reset_all_veteran_fitbit_device_records.rake:8]: @mchaffee-tw this is the guardrails for not running this in prod

File: modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records.rb (hunk 1 above)
Comment (yonidoronpeters) [reply modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records.rb:9]: see my other comment

**Pull request merged**